Documentation/install/upgrade-1.1.rst

```diff
@@ -0,0 +1,26 @@
+Upgrading to the Cilium 1.1 series
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Upgrading to Cilium 1.1.x from Cilium 1.1.y
+"""""""""""""""""""""""""""""""""""""""""""
+
+Set the version to the desired release per :ref:`upgrade_version`.
+
+Upgrading to Cilium 1.1.x from Cilium 1.0.y
+"""""""""""""""""""""""""""""""""""""""""""
+
+#. Follow the guide in :ref:`err_low_mtu` to update the MTU of existing
+   endpoints.
+
+#. :ref:`upgrade_cm`.
+
+   New options in Cilium 1.1:
+
+   * ``legacy-host-allows-world``: This is recommended to be set to false. For
+     more information, see :ref:`host_vs_world`.
+   * ``sidecar-istio-proxy-image``
+
+   See the :git-tree:`example ConfigMap
+   <examples/kubernetes/templates/v1/cilium-cm.yaml>` for more details.
```

Comment:
These options no longer exist in the cilium-cm.yaml can the git-tree point to a specific branch?

Reply:
AFAICT the above options exist both in the v1.1 branch and v1.2/master.

`:git-tree` already points to the "current" branch, eg if you're browsing 1.1 it will point to the example configmap for 1.1. It's defined based on `SCM_WEB`:

https://github.com/cilium/cilium/blob/693c576a89aafc11cad649266bf7f1be563637bc/Documentation/conf.py#L78-L86

Reply:
Ah, there is `sidecar-http-proxy` which was deprecated in 1.1. I can mention that too.